app/src/main/java/com/ricardoeihara/quotesapp/model/Quote.kt

```diff
@@ -0,0 +1,7 @@
+package com.ricardoeihara.quotesapp.model
+
+data class Quote(
+    val id: String? = null,
```

Comment:
Our api returns `_id` and not just `id`
```suggestion
    val _id : String?,
```